components/Lever/Lever.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import AnimatedSprite from '../AnimatedSprite/AnimatedSprite';
+//import fountainLever from '../../sprites/fountainLever/fountainLeverCharacter';
+//import lever from '../../sprites/lever/leverCharacter';
+
+
+class Lever extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      character: props.character,
```

Comment:
Most of these do not need to be in state. There will end up being two events, onPress lever animates to down position, onPressOut lever animation to up position.